src/Marten/Events/EventStoreExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Baseline;
+using Marten.Linq;
+
+namespace Marten.Events
+{
+    public static class EventStoreExtensions
+    {
+        public static T AggregateTo<T>(this IMartenQueryable<IEvent> queryable, T state = null) where T : class
```

Comment:
I've added a `state` parameter to more closely match `AggregateStream`. 

1. Is that a good idea? 
2. If it is, then I couldn't find any `AggregateStream` tests that explicitly test `state`. Presumably that's taken care of in the `Build` related tests?

Reply:
That's poor. Just a quick test using the state parameter would still help here to prove that it's wired up.